plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogParser.java

```diff
@@ -119,7 +119,7 @@ private TransactionLogParser() {}
             .withResolverStyle(ResolverStyle.STRICT);
     public static final DateTimeFormatter JSON_STATISTICS_TIMESTAMP_FORMATTER = new DateTimeFormatterBuilder()
             .parseCaseInsensitive()
-            .appendValue(YEAR, 4, 10, SignStyle.NORMAL)
+            .appendValue(YEAR, 4, 10, SignStyle.EXCEEDS_PAD)
```

Comment:
The context: https://github.com/trinodb/trino/issues/22184

Reply:
can this change be separate commit (along with some test if any applicable)?